### ISSUE TITLE
Prebuilt binaries for macOS, Windows, Linux

### ISSUE THREAD

**Issue by Nexuist** [HackerPoet/MarbleMarcher#7]
Hello,

Following in the footsteps of #1, it would be nice to have prebuilt binaries (via GitHub releases) that prospective users could download to avoid having to clone and build the whole project. In addition, this could allow non-developers to be exposed to the primary concept behind the game, the fractal game engine, and give it a wider audience.

I'm not sure if this is where you intend to take the project, or if you would prefer to keep it as an open source demo of sorts, but I figured it might be worth starting the conversation.

**Comment by HackerPoet** [comment]
This may be the direction I head down eventually if I decide to make another game with the engine.  Right now, there's poor separation between the game and the engine/physics, as it was thrown together quickly rather than as a long term project.  I also wasn't expecting the popularity that it got.  If I make something else with this engine, I will also pull out the engine components to a library so it can be used in more applications.

**Comment by swedneck** [comment]
I would also like to see prebuilt executables available for download, as compilation is extremely daunting for most people.

**Comment by tomlister** [comment]
Yeah compiling for macOS is a bitch, spent like 3 hours trying to get it to work.

**Comment by tuxutku** [comment]
> Yeah compiling for macOS is a bitch, spent like 3 hours trying to get it to work.

you might want to use ``make -j$(nproc)`` next time

**Comment by tomlister** [comment]
It's not an issue with cores, it's that fact that clang isn't getting headers from cmake. I haven't been able to make it compile (wasted 3 hours).

**Comment by markedphillips** [comment]
I just wasted an hour doing the same. Sucks, I really wanted to play this. Actually, I really wanted to have this as a screensaver.

**Comment by tuxutku** [comment]
> I just wasted an hour doing the same. Sucks, I really wanted to play this. Actually, I really wanted to have this as a screensaver.

[you might have the problem with old libsfml packages](
https://github.com/HackerPoet/MarbleMarcher/issues/6#issuecomment-459969337)

**Comment by C-Kenny** [comment]
On the flip-side, having the game behind a bunch of build step encourages curious individuals to get their hands dirty. I do agree that debugging environment and build issues are potentially the least fun aspects of life though.
